packages/compass/package.json

```diff
@@ -149,6 +150,7 @@
     "test-renderer": "xvfb-maybe electron-mocha --no-sandbox --config ./.mocharc.renderer.js \"./src/app/**/*.spec.*\"",
     "check": "npm run typecheck && npm run lint && npm run depcheck",
     "webpack": "webpack-compass",
+    "postwebpack": "cp package.json build/assets/compass-application-package.json",
```

Comment:
Actually, I'll be checking with Dylan to see if we can integrate this with papertrail properly (just need to figure out how to manually set a submitter)